packages/main/src/SplitButton.js

```diff
@@ -211,11 +211,15 @@ const metadata = {
 	events: /** @lends sap.ui.webcomponents.main.SplitButton.prototype */ {
 		/**
 		 * Fired when the user clicks on the default action.
+		 * @event
+		 * @public
```

Comment:
The click event for the `ui5-button` is marked as `@native` - do we need this flag here as well?

Reply:
This is for the UI5WebC enablement in OpenUI5 (a.k.a Retrofit) topic for some mapping logic, so it's needed.